scripts/hid_configurator/configurator.py

```diff
@@ -263,38 +319,80 @@ def perform_dfu(dev, args):
     img_length = os.stat(dfu_image).st_size
     print('DFU image size: {} bytes'.format(img_length))
 
+    for i in range(3):
+        dfu_info = dfu_sync(dev, args)
+        if dfu_info is not None:
+            if dfu_info[0] != 0:
+                time.sleep(1)
```

Comment:
Maybe add comment to explain why we need sleep here

Reply:
Done